addons/mrp/views/stock_warehouse_views.xml

```diff
@@ -7,14 +7,14 @@
             <field name="inherit_id" ref="stock.view_warehouse"/>
             <field name="arch" type="xml">
                 <xpath expr="//field[@name='resupply_wh_ids']" position="before">
-                    <field name="manufacture_to_resupply" />
-                    <field name="manufacture_steps" attrs="{'invisible': [('manufacture_to_resupply', '=', False)]}" widget="radio"/>
+                    <field name="manufacture_to_resupply"/>
+                    <field name="manufacture_steps" attrs="{'invisible': [('manufacture_to_resupply', '=', False)]}" widget="radio" groups="stock.group_adv_location"/>
```

Comment:
In case that only multi-location is activate (but not the multi-warehouse). We will see the Resupply group with only the `manufacture_steps`. It is weird, no ?
Maybe it is more appropriate to add a group for Manufacturing after the group Resupply for `manufacture_steps` and let the `manufacture_to_resupply` in the same place ?

Reply:
I don't find it's weird, and I'm not sure to understand why it would be less weird to have an another group with only one field in it too ?

Reply:
IMO, the manufacturing step shouldn't be in the resupply group, I don't see the link between manufacturing steps and resupply. IMO, the manufacturing steps should be in a new group just below the shipment one.
But if you think it is ok like that, I understand, it still a fix after all.